### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by SebastienPeillet**
@pblottiere Did I forget something in the Readme ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,43 @@ or add the following arguments in a shortcut configuration :
...
+
+### Keyboard shorcuts
+
+Some keyboard shortcuts are available in SAMMO-Boat :
+
+- `Shift+S` : save all tables manually
+- `Shift+E` : add an entity in environment table 
+- `Shift+O` : add an entity in sighting table
+- `Shift+F` : open follower dialog to add entities in follower table
+- `Ctrl+Z` : Undo last operation
```
Comment (pblottiere) [inline README.md:54]:
```suggestion
- `Ctrl+Z` : undo last operation
```

File: README.md [hunk 2]
```diff
@@ -41,6 +41,43 @@ or add the following arguments in a shortcut configuration :
...
+- `Ctrl+Shift+Z` : Redo last canceled operation
```
Comment (pblottiere) [inline README.md:55]:
```suggestion
- `Ctrl+Shift+Z` : redo last canceled operation
```

File: README.md [hunk 3]
```diff
@@ -41,6 +41,43 @@ or add the following arguments in a shortcut configuration :
...
+
+To use Undo / Redo shorcut, you need to click on the table you want to change before pressing the shortcut.
+
+### Tools
+
+#### Save tool
+
+The save tool <img src="https://github.com/hytechimaging/sammo-boat/blob/main/images/pen.png?raw=true" height="30" width="30" /> can be use to save manually all tables.
```
Comment (pblottiere) [inline README.md:63]:
```suggestion
The save tool <img src="https://github.com/hytechimaging/sammo-boat/blob/main/images/pen.png?raw=true" height="30" width="30" /> can be used to manually save all tables.
```

File: README.md [hunk 4]
```diff
@@ -41,6 +41,43 @@ or add the following arguments in a shortcut configuration :
...
+
+#### Export tool
+
+The export tool <img src="https://github.com/hytechimaging/sammo-boat/blob/main/images/export.png?raw=true" height="30" width="30" /> allow to export all tables as csv into an output directory.
```
Comment (pblottiere) [inline README.md:67]:
```suggestion
The export tool <img src="https://github.com/hytechimaging/sammo-boat/blob/main/images/export.png?raw=true" height="30" width="30" /> allows to export all tables as csv into an output directory.
```

File: README.md [hunk 5]
```diff
@@ -41,6 +41,43 @@ or add the following arguments in a shortcut configuration :
...
+
+#### Merge tool
+
+The merge tool <img src="https://github.com/hytechimaging/sammo-boat/blob/main/images/merge.png?raw=true" height="30" width="30" /> can be use to merge two sessions into another.
```
Comment (pblottiere) [inline README.md:71]:
```suggestion
The merge tool <img src="https://github.com/hytechimaging/sammo-boat/blob/main/images/merge.png?raw=true" height="30" width="30" /> can be used to merge two sessions into another.
```

File: README.md [hunk 6]
```diff
@@ -41,6 +41,43 @@ or add the following arguments in a shortcut configuration :
...
+
+To use it, fulfill the merge form with the directory of each session to merge and the output directory. The output session will contain each entity of both session. In case of strict identical entity, only one entity is copied. 
+
+#### Sound replay
+
+When a new entity is added, SAMMO-Boat will capture a sound record. To play it again, you can do a right click on the table line and click on `Play`. A audio player will pop up.
```
Comment (pblottiere) [inline README.md:77]:
```suggestion
When a new entity is added, SAMMO-Boat will capture a sound record. To play it again, you can do a right click on the table line and click on `Play`. An audio player will pop up.
```

**Comment by SebastienPeillet** [comment]
@pblottiere Thanks for the review ! ;)

**Pull request merged**